src/keyszer/xorg.py

```diff
@@ -56,19 +56,16 @@ def get_xorg_context():
 
 
 def get_actual_window(window):
-    try:
-        # use _NET_WM_NAME string instead of WM_NAME to bypass (COMPOUND_TEXT) encoding problems
-        wmname  = window.get_full_text_property(_display.get_atom("_NET_WM_NAME"))
-        wmclass = window.get_wm_class()
-        # workaround for Java app
-        # https://github.com/JetBrains/jdk8u_jdk/blob/master/src/solaris/classes/sun/awt/X11/XFocusProxyWindow.java#L35
-        if (wmclass is None and wmname is None) or "FocusProxy" in wmclass:
-            parent_window = window.query_tree().parent
-            if parent_window:
-                return get_actual_window(parent_window)
-            return None
-
-        return window
-    # TODO: more specific rescue here
-    except Exception:
+    # use _NET_WM_NAME string instead of WM_NAME to bypass (COMPOUND_TEXT) encoding problems
+    wmname  = window.get_full_text_property(_display.get_atom("_NET_WM_NAME"))
+    wmclass = window.get_wm_class()
+    debug(f'### get_actual_window():\n\t{wmname = }\n\t{wmclass = }')
```

Comment:
not sure we need this logging

Reply:
Quite right. Forgot to pull that back out after testing. Removed.